astronomer/providers/amazon/aws/example_dags/example_aws_nuke.py

```diff
@@ -0,0 +1,42 @@
+import os
+from datetime import datetime, timedelta
+
+from airflow import DAG
+from airflow.operators.bash import BashOperator
+from airflow.operators.dummy import DummyOperator
+
+AWS_ACCESS_KEY_ID = os.environ.get("AWS_ACCESS_KEY_ID", "**********")
+AWS_SECRET_ACCESS_KEY = os.environ.get("AWS_SECRET_ACCESS_KEY", "***********")
+AWS_DEFAULT_REGION = os.environ.get("AWS_DEFAULT_REGION", "us-east-2")
+AWS_CONN_ID = os.environ.get("ASTRO_AWS_CONN_ID", "aws_default")
+EXECUTION_TIMEOUT = int(os.getenv("EXECUTION_TIMEOUT", 6))
+
+default_args = {
+    "execution_timeout": timedelta(hours=EXECUTION_TIMEOUT),
+}
+
+with DAG(
+    dag_id="example_aws_nuke",
+    start_date=datetime(2022, 1, 1),
+    schedule_interval=timedelta(hours=1),
```

Comment:
One question I have is how do we ensure that we do not trigger this DAG while our `master_dag` from integration test is still busy executing. Otherwise, it will try cleaning up resources being used by our Integration tests. Also, what if for our local development, we spawned up some resource and this tries to clean it up; we will have to spawn up the required resource again.

Does it make sense to reduce the schedule_interval to `daily`and run it in some non-working hours and probably some time before/after our master_dag execution is done?

Reply:
Wouldn't `schedule_interval=timedelta(hours=1)` means run it every hour? Can we please confirm on this? 
I think we need to provide a cron expression to run at a specified time like `schedule_interval="0 1 * * *"` to run at 1 AM GMT. 

Also, our master DAG sent an alert today at `12:47 AM GMT (6:17 AM IST)`. One some day, it might well exceed` 1 AM GMT`. So might make sense to consider running it at `2 AM GMT (7:30 AM IST)` instead to allow resources be kept until the master DAG has completed. But this might mean that in case we forget to delete resources on Friday evening IST, they may run until next morning (Saturday IST) time.

Reply:
The DAG would run at start_date + timedelta. A time delta(hours=1) means 1AM daily.  Please check the link https://stackoverflow.com/questions/36298756/how-to-work-correctly-airflow-schedule-interval

Ideally we need to prevent un-necessary costs as much as possible. So thinking of using 20:00 hrs GMT which is equivalent to resources shutting down at 1:30AM IST. I dont think anyone from our team works at that time.

Reply:
changed to crontab expression